tools/bouncer_worker/src/tasks/unityEditor.js

```diff
@@ -66,7 +69,12 @@ UnityHandler.generateAssetBundles = async (database, model, rid, logDir, process
 		unityLog,
 	];
 
+	if (config.repoLicense) {
+		process.env.REPO_LICENSE = config.repoLicense;
+	}
+
 	try {
+    process.env.REPO_INSTANCE_ID = crypto.randomBytes(16).toString("hex");
```

Comment:
considering you only need the instance ID if license is enabled, shouldn't it be inside L72?

Reply:
generating randomBytes with `crypto` is a bit excessive, just use [uuid](https://www.npmjs.com/package/uuid)

Reply:
I thought it might be too overkill to add a new package just for this, but I'll go ahead and do it that way.